src/Microsoft.DotNet.Build.Tasks/PackageFiles/CodeCoverage.targets

```diff
@@ -5,8 +5,8 @@
     Code coverage package versions go here and in the test-runtime-packages.config
   -->
   <PropertyGroup>
-    <OpenCoverVersion>4.6.476-rc</OpenCoverVersion>
-    <ReportGeneratorVersion>2.4.0</ReportGeneratorVersion>
+    <OpenCoverVersion>4.6.519</OpenCoverVersion>
```

Comment:
For the next update can you add conditions to these something like:

```
 <OpenCoverVersion Condition="'$(OpenConverVersion)'==''">4.6.519</OpenCoverVersion>
```

That way we can set version in corefx repo and only need to update build tools version when there is a reasonable milestone build or we require changes to our targets files. This is the approach we have taken already with test-runtime/project.json file as well, we override it in corefx repo.